### DISCUSSION TITLE
Radix icons seem to not be affected when going to the drawer

### DISCUSSION THREAD

**Discussion by Paraxdev** [mantinedev/mantine#77] [Q&A] [answered]
I have an issue where when I open the drawer the "@modulz/radix-icons" inside ActionIcon doesn't get disabled or grayed out they also can be clicked. ![Help needed](https://i.imgur.com/fVn0H0O.gif) (i don't know if i made something wrong or this is a bug)
code:
```javascript
          <Container className="jumbotron bg-cover text-white ubuntu">
              <Container className="container py-5 text-center">
                  <img src=""
                       style={{borderRadius: "50%", outline: "white 2", margin: "2%"}} width="200" height="200"  alt={""}/>
                      <Text size={"lg"} className="display-4 font-weight-bold " style={{fontSize: "xx-large"}}></Text>
                      <Text className="font-italic mb-0"></Text>
                      <Container className="btn-group btn-group-justified">
                          <ActionIcon variant="transparent" size={"lg"} onClick={() => redirect("")} ><GitHubLogoIcon /></ActionIcon>
                          <ActionIcon variant="transparent" size={"lg"} onClick={() => redirect("")} ><FileTextIcon /></ActionIcon>
                      </Container>
              </Container>
          </Container>
```

**Comment by rtivital** [comment, marked as answer]
This happens because of z-index collision (ActionIcon has higher z-index value in your rendering context), to avoid that render drawer inside portal – https://mantine.dev/core/drawer/#usage-with-portal

**Reply by Paraxdev** [reply]
Thanks! Works flawlessly now.

**Reply by cstrat** [reply]
Is there any reason why `Drawer` and `Modal` aren't by default rendered in a portal?
I've had to use `Portal` for every `Modal` to avoid the z-index issues like in this question.

**Reply by cstrat** [reply]
I am guessing anything where an `Overlay` comes into play would benefit from rendering in a `Portal`...

**Reply by rtivital** [reply]
Actually I cannot think of any reason why it should not be inside portal by default, I'll add this in next release
